coalib/bears/Bear.py

```diff
@@ -250,7 +250,7 @@ def execute(self, *args, **kwargs):
             return [] if result is None else list(result)
         except:
             self.warn(
-                "Bear {} failed to run. Take a look at debug messages (`-L "
+                "Bear {} failed to run. Take a look at debug messages (`-V "
                 "DEBUG`) for further information.".format(name))
```

Comment:
`-V DEBUG`? `-V` doesn't need an argument so we'll want to remove the `DEBUG` from the command proposal.

This is weird to phrase, was this understandable? :/